app/advancedfilters/advancedFilters.js

```diff
@@ -60,9 +60,14 @@ app.controller('AdvancedFiltersCtrl', function($scope, $rootScope, $routeParams,
     $scope.resetgpIds = function() {
       filteringService.initGpSet();
       filteringService.initGpID();
+      $scope.updateFilters();
+    }
```

Comment:
While you are in this file, would it be sensible to add the missing semi-colons? Or will all of them be done in the branch created for that purpose?